scripts/management/query_mgmt_db.py

```diff
@@ -267,7 +323,7 @@ def main():
         "--detailed_count",
```

Comment:
detailed_count could be added as a mode, and the logic below sets count true as well